src/nest_utils.jl

```diff
@@ -35,7 +35,7 @@ should return a value other than `nothing`.
 """
 function walk(f, fst::FST, s::State)
     stop = f(fst, s)
-    (stop != nothing || is_leaf(fst)) && return
+    (!isnothing(stop) || is_leaf(fst)) && return
```

Comment:
I don't think this is compatible with earlier versions of Julia so we need to use !== or ===